pkg/reporting/app.go

```diff
@@ -144,6 +144,14 @@ func getDownstreamInfo(appID string) (*types.DownstreamInfo, error) {
 		di.Cursor = deployedKotsKinds.Installation.Spec.UpdateCursor
 		di.ChannelID = deployedKotsKinds.Installation.Spec.ChannelID
 		di.ChannelName = deployedKotsKinds.Installation.Spec.ChannelName
+
+		if len(deployedKotsKinds.HelmCharts) > 0 {
+			for _, chart := range deployedKotsKinds.HelmCharts {
+				if chart.Spec.UseHelmInstall {
+					di.IsNativeHelm = true
```

Comment:
this just made me realize we allow releases with some charts that use replicated helm and others that use native helm simultaneously 🤯